### ISSUE TITLE
Need an S3 bucket to be used by channel-diff script

### ISSUE THREAD

**Issue by garbas** [NixOS/nixos-org-configurations#125]
In https://github.com/NixOS/nixos-search/pull/206 I created a script that outputs diff between 2 channels. I'd like to run this on a periodic schedile - daily - via github actions and store this json files somewhere (I thought of S3 bucket).

Finally I would consume this json files in the status.nixos.org frontend.

/Cc @edolstra

**Comment by FRidh** [comment]
You could have it store the artifact in GitHub, e.g. using https://github.com/actions/upload-artifact.

**Comment by garbas** [comment]
@FRidh yes, but then I this artifact is not accessible via some public URL and can not be reused from eg. search.nixos.org or status.nixos.org.
